setup.cfg

```diff
@@ -1,57 +1,3 @@
-[metadata]
-license_file = LICENSE
-name = flake8-bugbear
-version = attr: bugbear.__version__
-author = Łukasz Langa
-author_email = lukasz@langa.pl
-license = MIT
-description = A plugin for flake8 finding likely bugs and design problems in your program. Contains warnings that don't belong in pyflakes and pycodestyle.
-keywords =
-    flake8
-    bugbear
-    bugs
-    pyflakes
-    pylint
-    linter
-    qa
-url = https://github.com/PyCQA/flake8-bugbear
-long_description = file: README.rst
-classifiers =
-    Development Status :: 5 - Production/Stable
-    Environment :: Console
-    Framework :: Flake8
-    Intended Audience :: Developers
-    License :: OSI Approved :: MIT License
-    Operating System :: OS Independent
-    Programming Language :: Python
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.6
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3 :: Only
-    Topic :: Software Development :: Libraries :: Python Modules
-    Topic :: Software Development :: Quality Assurance
-project_urls =
-    Change Log = https://github.com/PyCQA/flake8-bugbear#change-log
-
-[options]
-py_modules = bugbear
-zip_safe = False
-install_requires = flake8 >= 3.0.0; attrs>=19.2.0
-python_requires = >=3.6
-test_suite = tests.test_bugbear
-
-[options.entry_points]
-flake8.extension = B = bugbear:BugBearChecker
-
-[options.extras_require]
-dev =
-    coverage
-    hypothesis
-    hypothesmith>=0.2
-    pre-commit
-
 [flake8]
```

Comment:
Can't wait for flake8 to support config in the toml

Reply:
My sense is that will take time...